pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -96,8 +96,22 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
                   <TableCell>Rhoncus aenean vel elit scelerisque mauris</TableCell>
                 </TableRow>
                 <TableRow>
-                  <TableHeader>UPL</TableHeader>
-                  <TableCell>Sit amet nisl suscipit adipiscing bibendum est ultricies</TableCell>
+                  <TableHeader>{t("Products")}</TableHeader>
+                  <TableCell>
+                    {_getComponent.data.embedded.nl.upl.map((product: string, idx: any) => (
+                      <a
+                        key={idx}
+                        href="http://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland"
+                        target={"_blank"}
+                      >
```

Comment:
Even though this works (nothing wrong with it either), we've been doing outbound links in another way within the project. For consistency I'd recommend updating this to:

```
<span onClick={() => open("url here")} > // NOTE: This is the span element you already have!
  <Link icon={<ExternalLinkIcon />} iconAlign="start">
    {product},
  </Link>
</span>
```